spacy/lang/fr/stop_words.py

```diff
@@ -1,30 +1,31 @@
 STOP_WORDS = set(
     """
 a à â abord afin ah ai aie ainsi ait allaient allons
-alors anterieur anterieure anterieures apres après as assez attendu au
-aucun aucune aujourd aujourd'hui aupres auquel aura auraient aurait auront
+alors anterieur anterieure anterieures antérieur antérieure antérieures
+apres après as assez attendu au
+aucun aucune aucuns aujourd aujourd'hui aujourd’hui aupres auquel aura auraient aurait auront
 aussi autre autrement autres autrui aux auxquelles auxquels avaient
 avais avait avant avec avoir avons ayant
 
 bas basee bat
 
-c' c’ ça car ce ceci cela celle celle-ci celle-là celles celles-ci celles-là celui
-celui-ci celui-là cent cependant certain certaine certaines certains certes ces
+c' c’ ça car ce ceci cela celle celle-ci celle-la celle-là celles celles-ci celles-la celles-là
+celui celui-ci celui-la celui-là cent cependant certain certaine certaines certains certes ces
 cet cette ceux ceux-ci ceux-là chacun chacune chaque chez ci cinq cinquantaine cinquante
 cinquantième cinquième combien comme comment compris concernant
 
-d' d’ da dans de debout dedans dehors deja delà depuis derriere
+d' d’ da dans de debout dedans dehors deja dejà delà depuis derriere
 derrière des desormais desquelles desquels dessous dessus deux deuxième
-deuxièmement devant devers devra different differentes differents différent
+deuxièmement devant devers devra different differente differentes differents différent
 différente différentes différents dire directe directement dit dite dits divers
 diverse diverses dix dix-huit dix-neuf dix-sept dixième doit doivent donc dont
-douze douzième du duquel durant dès désormais
+douze douzième du duquel durant dès déja déjà désormais
 
-effet egale egalement egales eh elle elle-même elles elles-mêmes en encore
+effet egalement eh elle elle-meme elle-même elles elles-memes elles-mêmes en encore
 enfin entre envers environ es ès est et etaient étaient etais étais etait était
-etant étant etc été etre être eu eux eux-mêmes exactement excepté
+etant étant etc ete été etre être eu eux eux-mêmes exactement excepté également
 
-fais faisaient faisant fait façon feront font
+fais faisaient faisant fait facon façon feront font
```

Comment:
- removed "egale" and "egales" (why only the feminine version of this word were there escapes me; equality is an important topic in many texts analyzed with spacy);
- added "également" (with accent);
- added "ete" (été sans accents);
- added "facon" (without ç);
- added "elle/s-meme" as well as "soi-meme" and "toi-même" (for the sake of coherence, if "lui-meme" is there);